test/api/league_test.py

```diff
@@ -89,22 +88,28 @@ def test_post_returns_400_if_invalid_rating_scheme_is_specified(self):
         self.assertEqual(response.data, '"Invalid rating scheme %s"\n' % self.args_mock.return_value['rating_scheme'])
 
     def test_post_new_league_when_created_successfully(self):
-        self.create_test_league()
-        self.mock_function_in_setup('app.api.league.create_league', return_value=self.league)
+        self.create_test_user()
```

Comment:
@sourlows Mocking create_league() for this test bugged out when I changed stuff in src/main.py (??). I'm iffy on the way I changed it, but I couldn't find a way to make it correctly test post as written before.